pkg/defaults/policies/files/severity_high_privileged.json

```diff
@@ -0,0 +1,59 @@
+{
+  "id": "b1df1abb-e5a5-4ff7-98fe-1d28a22b55d8",
+  "name": "Privileged Container(s) with Important and Critical CVE(s)",
```

Comment:
```suggestion
  "name": "Privileged Container(s) with Important and Critical fixable CVE(s)",
```

Reply:
RH/IBM style in text is to not use (s) to indicate that a noun can be singular or plural. Not sure if there is a different convention for GUI text (since there's less screen space, maybe we need to use it), but I thought I'd point it out.

> Do not use the letter s in parentheses (s) to indicate that a noun can be singular or plural. Some languages form plural nouns differently than English, and the construction (s) can cause translation problems. Instead, use the plural form or, if it is important to indicate both singular and plural options, use the phrase one or more.

Reply:
PatternFly UX writing guidelines agree

https://www.patternfly.org/v4/ux-writing/punctuation#parentheses

> Do not use parentheses to indicate a possible plural of something, like "Account(s)." If a user can select one thing or multiple things, use the plural form.